_11ty/plugins/markdown-plugins.js

```diff
@@ -38,7 +38,8 @@ module.exports = function syntaxHighlighting(eleventyConfig) {
 
                 // then replace all -- with -
                 .replace(/-+/gu, "-");
-        }
+        },
+        uniqueSlugStartIndex: 1
```

Comment:
This isn't necessary since it's the default in v6.0.0, but might be safe to set it explicitly.